### ISSUE TITLE
"Only persisted queries" documentation is outdated

### ISSUE THREAD

**Issue by trancikk** [ChilliCream/graphql-platform#7729]
### Product

Hot Chocolate

### Version

14.1.0

### Link to minimal reproduction

none

### Steps to reproduce

Configure gql server to allow only persisted queries using documentation
here is  the link:
https://chillicream.com/docs/hotchocolate/v14/performance/persisted-operations#blocking-regular-operations

it mentions using OnlyAllowPersistedOperations, like this 
```

builder.Services
    .AddGraphQLServer()
    // Omitted for brevity
    .ModifyRequestOptions(o => o.OnlyAllowPersistedOperations = true);
```

### What is expected?

Code is at least compilable 

### What is actually happening?

method OnlyAllowPersistedOperations  doesn't exist 

Building gives an error
`Error CS1061 : 'RequestExecutorOptions' does not contain a definition for 'OnlyAllowPersistedOperations' and no accessible extension method 'OnlyAllowPersistedOperations' accepting a first argument of type 'RequestExecutorOptions' could be found (are you missing a using directive or an assembly reference?)`
![Image](https://github.com/user-attachments/assets/ba5f70a8-b9e3-47ba-bd91-8b2e17d086a7)

### Relevant log output

```shell

```

### Additional context

It seems it has been reworked in the https://github.com/ChilliCream/graphql-platform/commit/99a571abf96d8a737ee4c4c3af76992a3a887008#diff-e7261c7e2cdbca7a1f0ea9b90c2daac10b929e94880e68a35eb25202347980f5L13 , however I would assume it was supposed to go to the the next version?

**Comment by trancikk** [comment]
@glen-84  thank you for the quick update!
You guys might want to update migration doc as well
https://github.com/ChilliCream/graphql-platform/blob/80046433304de154819eb42aad2d41588aba2ef6/website/src/docs/hotchocolate/v15/migrating/migrate-from-13-to-14.md?plain=1#L194

Its still using an old name
![Image](https://github.com/user-attachments/assets/1dc92c0c-941e-4a84-926c-8250e9501fc0)

**Comment by glen-84** [comment]
@trancikk Thanks, fixed in #7737.
